krcal/core/fit_lt_functions_test.py

```diff
@@ -8,7 +8,11 @@
 from pytest                import approx
 from pytest                import warns
 
-from invisible_cities.core import fit_functions as fitf
+from hypothesis            import given
+from hypothesis.strategies import integers, floats
+
+from invisible_cities.core               import fit_functions as fitf
+from invisible_cities.core.testing_utils import float_arrays
```

Comment:
fix the import and im ready to approve! :)

Reply:
Ah that's true! I forgot to remove them. Now I've done it.
Thanks.